openssl/src/dh.rs

```diff
@@ -21,7 +42,17 @@ impl DhRef {
     to_der!(ffi::i2d_DHparams);
 }
 
+
+// TODO Implement overloaded DH initialization for equivalent methods in 1.1.x
 impl Dh {
+    
+    /// This method will set the c pointers for the DH object, p, q, and g. If the parameters have
+    /// not been set, they will default to NULL.
```

Comment:
The second sentence doesn't seem super relevant - we are setting them.

Reply:
@sfackler When I read the code, I read it as we're taking values in, regardless of what they are, and setting them as the ptrs. Do we not inherit the behavior of defaulting to `NULL` when someone calls this function and provides nothing for the actual values for `p`, `g`, and `q`?

Reply:
The user can't provide nothing for those values.